recipe/meta.yaml

```
@@ -30,6 +30,7 @@ test:
 about:
   home: https://github.com/benediktschmitt/py-filelock
   license: Public Domain
+  license_file: LICENSE.rst
```

Should it be just ``LICENSE``?
```suggestion
  license_file: LICENSE
```

---

The file in version `3.0.10` is `LICENSE.rst`, but you are right that since then they have changed it to `LICENSE`. 👍

---

ah ok then! :grimacing: 
Thanks for double checking it